crates/next-core/js/src/overlay/internal/ReactDevOverlay.tsx

```diff
@@ -3,6 +3,7 @@ import * as React from "react";
 import type { Issue } from "@vercel/turbopack-runtime/types/protocol";
 
 import * as Bus from "./bus";
+import { TYPE_REACT_ERROR } from "./bus";
```

Comment:
Nit: this is the only value we import this way, while we refer to the others as `Bus.*`.

Reply:
fixed